cli/replicaset/cconfig.go

```diff
@@ -22,6 +22,11 @@ var (
 
 	//go:embed lua/cconfig/promote_election.lua
 	cconfigPromoteElectionBody string
+
+	//go:embed lua/cconfig/bootstrap_vshard_body.lua
+	cconfigBootstrapVShardBody string
+
+	cconfigGetShardingRolesBody = "return require('config'):get().sharding.roles"
```

Comment:
I think we should add roles here for cartridge and centralized config (and print them in status).

https://github.com/tarantool/tt/blob/055990d8694a9d665d587a69df794a152ec711d8/cli/replicaset/instance.go#L7-L23

Reply:
Let's do it in the separate task. https://github.com/tarantool/tt/issues/849